### ISSUE TITLE
A11y_code.visualstudio.com_DocsLanguages_ Java_Usability: Incomplete instructions provided for the video controls.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-docs#2782]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: https://code.visualstudio.com
Windows Version: Windows10
Browser Version: 
Microsoft Edge 42.17134.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1

**Repro Steps:**
Navigate to the url: [https://code.visualstudio.com/docs/languages/java](https://code.visualstudio.com/docs/languages/java)
follow below steps:
1.	Launch the application Url: [https://code.visualstudio.com](https://code.visualstudio.com)
2.	Navigate to "Docs" link.
3.	From left navigation, select "Languages".
4.	Navigate to "Java" link and activate it.
5.	Start screen reader.
6.	Navigate to the video content and then navigate to the video controls
7.	Listen to how they are announced.

**Actual:**
When screen reader users navigates to the video controls to interact with them, incorrect instructions are announced for each of the controls i.e. for Pause button it is announced as “Pause button space pauses the current media selection.

**Expected:**
The video controls should be announced with correct instructions for the screen reader users.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If correct instructions are not announced to the screen reader users then the users might not be able to access them.

**Attachment for Reference:**
[Code.visualstudio.com_Docs Setup_ Java_Usability Incomplete instructions for Video controls.pptx](https://github.com/microsoft/vscode-docs/files/3314237/Code.visualstudio.com_Docs.Setup_.Java_Usability.Incomplete.instructions.for.Video.controls.pptx)

**Comment by gregvanl** [comment]
Closing for now since Usability issues, labelled A11yUsable, are not currently required for Grade C.

**Comment by SwatiTiwari-TCS** [comment]
"The issue is still reproducing can you please open the issue".

**Comment by Ashutoshsahoo164** [comment]
GitHubTags: #P1_A11y_code.visualstudio.com_Apr2020;
